_emails/us/california/san_mateo_county.md

```diff
@@ -0,0 +1,69 @@
+---
+title: San Mateo County, CA
+permalink: "/sanmateocounty"
```

Comment:
maybe /san-mateo-county?

have been on the fence about hyphen vs no hyphen, but i think here it makes sense to use it